api/v1alpha1/releaselink_types.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ReleaseLinkSpec defines the desired state of ReleaseLink
+type ReleaseLinkSpec struct {
+	// DisplayName refers to the name of the Release Link to link a user and managed workspace together.
+	DisplayName string `json:"displayName"`
+
+	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
+	// +required
+	// Application to add the component to
```

Comment:
Would it make sense to move the description to the first line of the doc block? That way you don’t go through the kubebuilder tags before seeing what it’s actually for. Easier to read IMHO.

Reply:
I assumed description last was best practice because of https://github.com/redhat-appstudio/application-service/blob/main/api/v1alpha1/component_types.go#L82 if that is not the case though, I can change it. I don't have a strong opinion, but you are probably right about easier to read